packages/search-in-workspace/src/browser/search-in-workspace-widget.tsx

```diff
@@ -248,8 +248,10 @@ export class SearchInWorkspaceWidget extends BaseWidget implements StatefulWidge
 
     protected onUpdateRequest(msg: Message): void {
         super.onUpdateRequest(msg);
-        ReactDOM.render(<React.Fragment>{this.renderSearchHeader()}{this.renderSearchInfo()}</React.Fragment>, this.searchFormContainer);
-        this.onDidUpdateEmitter.fire(undefined);
+        if (this.renderSearchInfo()) {
```

Comment:
please extract a result of `this.renderSearchInfo()` to a local variable, otherwise you render it twice